src/Common/src/System/Security/Cryptography/Asn1/Pbkdf2Params.xml.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Security.Cryptography.Asn1
     [StructLayout(LayoutKind.Sequential)]
     internal partial struct Pbkdf2Params
     {
-        private static byte[] s_defaultPrf = { 0x30, 0x0C, 0x06, 0x08, 0x2A, 0x86, 0x48, 0x86, 0xF7, 0x0D, 0x02, 0x07, 0x05, 0x00 };
+        private static readonly byte[] s_defaultPrf = { 0x30, 0x0C, 0x06, 0x08, 0x2A, 0x86, 0x48, 0x86, 0xF7, 0x0D, 0x02, 0x07, 0x05, 0x00 };
```

Comment:
All of the .xml.cs files are generated; and this (and any other changes) will just get undone the next time the generator is run.

Reply:
> All of the .xml.cs files are generated; and this (and any other changes) will just get undone the next time the generator is run.

Where are the .xml files, and how are the generators run?

Reply:
Next to each of the .xml.cs files.   https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Security/Cryptography/Asn1/asn.xslt is the generator, it's run by desktop msbuild only when any of the .xml files is newer than their .xml.cs counterpart.  (When the generator was added core's msbuild didn't have XSLT support, and I don't know if that's changed yet)

Reply:
Thanks.  I updated the xslt and regenerated the .xml.cs files (I think I got them all).

Since we actively discourage using the desktop msbuild with corefx, we should really try to switch over to make this work with `dotnet msbuild`. @rainersigwald, @krwq, is this still blocked?

Reply:
I took a quick look about a year ago and it wasn't trivial to turn XslTransform on: https://github.com/microsoft/msbuild/issues/3368#issuecomment-394777235. It looks like `AssemblyBuilder.Save` is still not exposed, so I'd expect that to still be a problem.

But looking more closely, those errors seem to all be in tests. Maybe we could rewrite those tests or just disable the ones that use non-Core functionality. I marked that issue to take a look in the 16.4 timeframe.

Reply:
Thanks, @rainersigwald.